pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -255,6 +255,84 @@ static __always_inline enum parse_result read_with_remainder(kafka_response_cont
     return RET_DONE;
 }
 
+static __always_inline enum parse_result read_with_remainder_s16(kafka_response_context_t *response, pktbuf_t pkt,
```

Comment:
In the Fetch v12 PR, there is a version of this which avoids the copy/paste, so depending on which one is merged first we can resolve the conflicts in the favor of that version. So this version should be OK for this PR.

Reply:
I removed this as I merged the Fetch v12 PR